.github/workflows/scripts/check_for_new_files.js

```diff
@@ -0,0 +1,77 @@
+module.exports = {
+  getAddedFiles: ({ github, context, core }) => {
+    const {
+      issue: { number: issue_number },
+      repo: { owner, repo }
+    } = context;
+
+    // Use the Github API to query for the list of files from the PR
+    github
```

Comment:
Can we use `github.rest`? I think that is easier to read and is consistent with the other API call you make below. You'd have to make this function async tho, but I prefer `await`s then `.then`s.

Reply:
I think we could, but I wanted to use `github.paginate` because it makes it easier to grab all the files and filter them inline since the list of files for a PR might require pagination: https://octokit.github.io/rest.js/v19#pagination

If you think changing it to `github.rest` would be easier than I can do that too, otherwise I could add a comment to explain why I'm using `paginate`

Reply:
That makes sense to me, I feel like code is self explanatory then